.travis.yml

```diff
@@ -37,26 +45,13 @@ before_install: |
   nvm use 7.2.1
   npm config set python `which python`
   if [ "$TRAVIS_OS_NAME" == "osx" ]; then
-    brew update; 
-    brew install openssl readline 
-    brew outdated pyenv || brew upgrade pyenv 
-    brew install pyenv-virtualenv
-    brew install pyenv-virtualenvwrapper
-    pyenv install $PYTHON 
-    pyenv virtualenv $PYTHON MYVERSION
-    source ~/.pyenv/versions/MYVERSION/bin/activate
-    python --version 
-    python -c 'import sys;print(sys.version)' 
-    python -c 'import sys;print(sys.executable)' 
+    pyenv install $PYTHON
+    pyenv global $PYTHON
   fi
 install:
-# we have this here so we can see where python is installed and hardcode in our tests
-# else when running npm test, the python version picked is completely different :(
-  - python --version 
-  - python -c 'import sys;print(sys.executable)'
   - pip install -r requirements.txt
```

Comment:
Probably want `pip install --upgrade -r requirements.txt` for when unpinned dependencies change.

Reply:
fixed